src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -404,7 +404,7 @@ export class EditorPart extends Part implements IEditorPart, IEditorGroupService
 			}
 
 			// Focus (unless prevented)
-			const focus = !options || !options.preserveFocus;
+			const focus = !options || options.preserveFocus;
```

Comment:
Ok, I understand that.
If the `options` object is undefined it's going to focus the thing anyway - how should we handle this situation?